pages/index.md

```diff
@@ -1,14 +1,16 @@
 title: The PlasmaPy Project
 hidetitle: True
 
-<!-- Survey Card -->
+<!-- Announcement Card -->
 <div class="feature-row" style="margin: 24px 0">
     <div class="feature-column" style="width: 100%; padding: 0 10%">
         <a class="feature-link" href="https://docs.google.com/forms/d/e/1FAIpQLSeug7Wg1wWZdO10qV1X6dsOq1hn7kq3x8EXoDDjNYZ74ncuug/viewform">
```

Comment:
Just noticed this is still linking to the Survey.  I'll do a quick patch and then merge this PR.

Reply:
Good catch!  And thanks!